### PR TITLE
Update config-plugins dep version

### PR DESCRIPTION
**Pull request by kbrandwijk**
# Checklist

- [X] I've added an entry to [CHANGELOG.md](https://github.com/expo/sentry-expo/blob/main/CHANGELOG.md#master) if necessary.

# Why

`@expo/config-plugins` version needs a bump to prevent warnings on SDK 46.

# How

Updated to `~5.0.0`

# Test Plan

Tested locally.

### CODE REVIEW

**Comment by kbrandwijk** [comment]
/changelog-entry chore Bump `@expo/config-plugins` to `~5.0.0`

**Comment by kbrandwijk** [comment]
Looks like there's more problems:
image
image

**Review by brentvatne — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -37,13 +37,13 @@
...
+    "@expo/config-plugins": "~5.0.0",
+    "@expo/config-types": "^46.0.1",
     "@expo/spawn-async": "^1.6.0",
-    "@sentry/browser": "^7.7.0",
-    "@sentry/integrations": "^7.7.0",
-    "@sentry/react-native": "^4.1.3",
-    "@sentry/types": "^7.7.0",
+    "@sentry/browser": "7.7.0",
+    "@sentry/integrations": "7.7.0",
+    "@sentry/react-native": "4.1.3",
```
Comment (SimenB) [inline package.json:45]: this uses 7.6.0

Comment (SimenB) [reply package.json:45]: #277

**Pull request merged**